### DISCUSSION TITLE
[Feature] Delete already backed up items from mobile

### DISCUSSION THREAD

**Discussion by PilaScat** [immich-app/immich#13260] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add the ability to delete items already backed up with one click to clear memory quickly

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thank you. Duplicate of #165
